chains/gravity/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://explorer.gravity.xyz/api
+    family: blockscout
+    name: Gravity Alpha Mainnet Explorer
+    url: https://explorer.gravity.xyz
+blocks:
+  confirmations: 1
+  estimateBlockTime: 1
+  reorgPeriod: 5
+chainId: 1625
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: Gravity Alpha Mainnet
```

Comment:
```suggestion
displayName: Gravity Alpha
```

Reply:
Or set a displayNameShort field

Reply:
I'll set `displayNameShort` to `Gravity`